components/sponsors/GoldSponsors.tsx

```diff
@@ -0,0 +1,42 @@
+import { goldSponsors } from './GoldSponsorsList';
+
+interface GoldSponsorsProps {
+  className: string;
+  showSupportBanner: boolean;
+};
+
+/**
+ * This component displays Gold Sponsors.
+ * @param {GoldSponsorsProps} props - The props for GoldSponsors component.
+ * @param {string} props.className - Additional CSS classes for styling.
+ * @param {boolean} props.showSupportBanner - Indicates whether support banner should be displayed.
+ */
+export default function GoldSponsors({
+  className = '',
+  // eslint-disable-next-line no-unused-vars, unused-imports/no-unused-vars
+  showSupportBanner = true
```

Comment:
You can remove this prop if it is not being used anywhere inside the component, instead of disabling the eslint.

Reply:
Done